packages/devtools_app/lib/src/flutter/table.dart

```diff
@@ -699,15 +655,62 @@ class _TableState<T> extends State<_Table<T>> with AutoDisposeMixin {
       setState(() {
         final selection = oldWidget.selectionNotifier.value;
         if (selection.scrollIntoView) {
-          final newPosition = selection.displayRow * defaultRowHeight;
-          // TODO(terry): Optimize selecting row that's in the viewport
-          //              otherwise we jumpTo.
-          scrollController.jumpTo(newPosition);
+          final selectedDisplayRow = selectionRowNumber(selection.node);
+
+          // Compute total rows above immediate parent.
+          var parentNode = selection.node.parent;
```

Comment:
parentNode doesn't seem to be used? I'm missing where we are computing the total rows about the immediate parent.

Reply:
Code was cleaned up and removed.